pkgs/development/python-modules/bravado-core/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, buildPythonPackage, fetchFromGitHub, python-dateutil, jsonref, jsonschema,
+  pyyaml, simplejson, six, pytz, msgpack, swagger-spec-validator, rfc3987,
+  strict-rfc3339, webcolors, mypy-extensions, jsonpointer, idna, pytest, mock,
+  pytest-benchmark, isPy27, enum34 }:
+
+buildPythonPackage rec {
+  pname = "bravado-core";
+  version = "5.16.1";
+
+  src = fetchFromGitHub {
+    owner = "Yelp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0r9gk5vkjbc407fjydms3ik3hnzajq54znyz58d8rm6pvqcvjjpl";
+  };
+
+  checkInputs = [
+    mypy-extensions
+    pytest
+    mock
+    pytest-benchmark
+  ];
+
+  checkPhase = ''python -m pytest  --benchmark-skip'';
```

to support pypy and other alternative interpreters
```suggestion
  checkPhase = ''pytest  --benchmark-skip'';
```

---

👍